sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/README.md

```diff
@@ -94,57 +94,57 @@ npx cross-env AZURESTACKHCI_SUBSCRIPTION_ID="<azurestackhci subscription id>" AZ
-[virtualharddiskscreateorupdatesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksCreateOrUpdateSample.js
-[virtualharddisksdeletesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksDeleteSample.js
-[virtualharddisksgetsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksGetSample.js
-[virtualharddiskslistallsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksListAllSample.js
-[virtualharddiskslistsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksListSample.js
-[virtualharddisksupdatesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualHardDisksUpdateSample.js
-[virtualmachineinstancescreateorupdatesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesCreateOrUpdateSample.js
-[virtualmachineinstancesdeletesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesDeleteSample.js
-[virtualmachineinstancesgetsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesGetSample.js
-[virtualmachineinstanceslistsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesListSample.js
-[virtualmachineinstancesrestartsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesRestartSample.js
-[virtualmachineinstancesstartsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesStartSample.js
-[virtualmachineinstancesstopsample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesStopSample.js
-[virtualmachineinstancesupdatesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4-beta/javascript/virtualMachineInstancesUpdateSample.js
+[galleryimagescreateorupdatesample]: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/azurestackhci/arm-azurestackhci/samples/v4/javascript/galleryImagesCreateOrUpdateSample.js
```

Comment:
how is the arm-azure-stackhci samples changes is relevant to communication-call-automation GA ? the same for other arm related packages. if, it's not relevant, please undo the changes. Thank you.